iOSClient/Share/NCShareUserCell.swift

```diff
@@ -50,36 +50,40 @@ class NCShareUserCell: UITableViewCell, NCCellProtocol {
         guard let tableShare = tableShare else {
             return
         }
-        self.accessibilityCustomActions = [UIAccessibilityCustomAction(
-            name: NSLocalizedString("_show_profile_", comment: ""),
-            target: self,
-            selector: #selector(tapAvatarImage))]
-
+        contentView.backgroundColor = NCBrandColor.shared.secondarySystemGroupedBackground
         labelTitle.text = tableShare.shareWithDisplayname
-        labelTitle.textColor = .label
+        labelTitle.textColor = NCBrandColor.shared.label
+        labelCanEdit.text = NSLocalizedString("_share_permission_edit_", comment: "")
+        labelCanEdit.textColor = NCBrandColor.shared.iconColor
         isUserInteractionEnabled = true
-        labelQuickStatus.isHidden = false
-        imageDownArrow.isHidden = false
+        switchCanEdit.isHidden = true//false
+        labelCanEdit.isHidden = true//false
         buttonMenu.isHidden = false
-        buttonMenu.accessibilityLabel = NSLocalizedString("_more_", comment: "")
         imageItem.image = NCShareCommon.shared.getImageShareType(shareType: tableShare.shareType)
-
         let status = NCUtility.shared.getUserStatus(userIcon: tableShare.userIcon, userStatus: tableShare.userStatus, userMessage: tableShare.userMessage)
         imageStatus.image = status.onlineStatus
         self.status.text = status.statusMessage
-
+        
+        if CCUtility.isAnyPermission(toEdit: tableShare.permissions) {
+            switchCanEdit.setOn(true, animated: false)
+        } else {
+            switchCanEdit.setOn(false, animated: false)
+        }
+        
         // If the initiator or the recipient is not the current user, show the list of sharees without any options to edit it.
         if tableShare.uidOwner != userId && tableShare.uidFileOwner != userId {
-            isUserInteractionEnabled = false
-            labelQuickStatus.isHidden = true
-            imageDownArrow.isHidden = true
-            buttonMenu.isHidden = true
+           isUserInteractionEnabled = false
```

Comment:
No need to format the existing NC code as it will be considered as our customization.